openvpn/pia/configure-openvpn.sh

```diff
@@ -26,5 +26,19 @@ curl -sSL "${baseURL}/${PIA_OPENVPN_CONFIG_BUNDLE}.zip" -o "$tmp_file"
 echo "Extract OpenVPN config bundle into PIA directory $VPN_PROVIDER_HOME"
 unzip -qjo "$tmp_file" -d "$VPN_PROVIDER_HOME"
 
+# Normalize *.ovpn files to be lower case and replace sequential non-alphanumeric with an underscore
+pushd "$VPN_PROVIDER_HOME" > /dev/null
+for file in *.ovpn; do
+    normalized="$(echo "$file" | awk '{gsub(/[^a-zA-Z\.]+/, "_") ; print tolower($0)}')"
```

Comment:
Just realized now that I'm laying in bed, I forgot to include numbers, so this will basically only keep alpha characters. I'll try to remember to update that in the morning after I get all my morning stuff taken care of.

Reply:
reminder for you

Reply:
Thanks! I've just updated the pattern to include numbers.